Library/EnvelopeFactory.ts

```diff
@@ -152,6 +152,11 @@ class EnvelopeFactory {
         exception.severityLevel = Contracts.SeverityLevel.Error;
         exception.measurements = telemetry.measurements;
         exception.exceptions = [];
+        if (!isNaN(telemetry.severity)) {
+            exception.severityLevel = telemetry.severity;
+        } else {
+            exception.severityLevel = Contracts.SeverityLevel.Error;
+        }
```

Comment:
Can this be moved to where `exception.severityLevel` is initially set?

Reply:
I missed that line :p I'm fixing it